docs/hydra/configure-deploy.mdx

```diff
@@ -101,11 +101,11 @@ Migration successful!
 docker run -d \
   --name ory-hydra-example--hydra \
   --network hydraguide \
-  -p 9000:4444 \
-  -p 9001:4445 \
+  -p 5444:4444 \
```

Comment:
Maybe add a small section *why* we remap? Then we can also keep it in the 9000 style :) I think we use this style elsewhere as well, here for example: https://www.ory.sh/run-oauth2-server-open-source-api-security/

Reply:
Yep. I'll add a small description. I checked the documentation and found that it's only one case